bayesian/utils.py

```diff
@@ -32,9 +32,13 @@
 # TODO remove hardcoded gremlin_url when moving to Production This is just
 #      a stop-gap measure for demo
 
-gremlin_url = "http://{host}:{port}".format(
-    host=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_HOST", "localhost"),
-    port=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_PORT", "8182"))
+GREMLIN_SERVER_URL_REST = "http://{host}:{port}".format(
+    host=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_HOST"),
+    port=os.environ.get("BAYESIAN_GREMLIN_HTTP_SERVICE_PORT"))
+
+LICENSE_SCORING_URL_REST = "http://{host}:{port}".format(
+    host=os.environ.get("LICENSE_SERVICE_HOST"),
+    port=os.environ.get("LICENSE_SERVICE_PORT"))
```

Comment:
These environment variables are not defined anywhere (?) 

You could use: `F8A_LICENSE_ANALYSIS_SERVICE_HOST` and `F8A_LICENSE_ANALYSIS_SERVICE_PORT`. Those two will be provided automatically by OpenShift.

Reply:
These two variables (LICENSE_SERVICE_HOST,LICENSE_SERVICE_PORT) are defined as environment variables in devcluster